src/components/Avatar/Avatar.module.css

```diff
@@ -0,0 +1,40 @@
+.shared {
+  @apply rounded-full inline-block;
+}
+
+/* Avatar Size */
+
+.size-sm {
+  @apply w-xl h-xl;
+}
+.size-md {
+  @apply w-xxl h-xxl;
+}
+.size-lg {
+  /* {w|h}-xxxl is 56px not 48px */
+  width: 48px;
+  height: 48px;
```

Comment:
apply these classes in `tailwind.config` file and access them through `@apply` directive